modules/core/src/main/java/org/apache/synapse/core/axis2/ProxyService.java

```diff
@@ -668,19 +668,21 @@ public AxisService buildAxisService(SynapseConfiguration synCfg, AxisConfigurati
 
             for (PolicyInfo pi : policies) {
 
-                String key = pi.getPolicyKey();
+                String policyKey = pi.getPolicyKey();
                 Policy policy = null;
 
-                synCfg.getEntryDefinition(key);
-                Object o = synCfg.getEntry(key);
-                if (o == null) {
-                    handleException("NoSecurity Policy found from key " + key + " in Axis2 service \"" + name + "\"");
+                synCfg.getEntryDefinition(policyKey);
+                Object policyEntry = synCfg.getEntry(policyKey);
+                if (policyEntry == null) {
+                    handleException("Security Policy Entry not found for key: " + policyKey +
```

Comment:
line wrapping needs to happen before an operator, shall we correct in all places?

Reply:
In Synapse almost all the places it is done in the other way. I don't want to break the consistency or I don't want to change all the places :)